spec/controllers/api/v1/animals_controller_spec.rb

```diff
@@ -44,21 +56,22 @@
   end
 
   describe "GET/show" do
-    it "should return an individual animal" do
+    it "should return an individual animal and its associated reviews" do
```

Comment:
I would suggest that you break this into two tests
`it "should return an individual animal" do`
and
`it "should return the reviews for an animal"`

This ensures that your animal will display without any reviews being present, and that when your animal renders, it brings its reviews with it.